.github/workflows/main.yml

```diff
@@ -225,4 +225,4 @@ jobs:
           path: ./**/*.so
           key: ${{ runner.os }}-builds-${{ github.sha }}
       - name: Test Rust client
-        run: pnpm clients:rust:test
+        run: RUST_LOG=error pnpm clients:rust:test
```

Comment:
nit: Was this intended?

Reply:
Yes – this is part of my attempt to minimize the chance of `RpcError(DeadlineExceeded)`. Not sure if makes sense though. My rationale was to avoid "wasting" time writing the log.